gatling-core/src/main/scala/io/gatling/core/session/el/ElCompiler.scala

```diff
@@ -366,7 +391,11 @@ final class ElCompiler private extends RegexParsers {
 
   private def currentDate: Parser[ElPart[Any]] = "currentDate(" ~> DateFormatRegex <~ ")" ^^ (format => CurrentDateTimePart(new SimpleDateFormat(format)))
 
-  private def nonSessionObject: Parser[ElPart[Any]] = currentTimeMillis | currentDate
+  private def randomSecuredUuid: Parser[ElPart[Any]] = "randomSecuredUuid()" ^^ (_ => RandomSecureUUID)
```

Comment:
My bad, it should be `secure`, not `secured`. Sorry for the original typo.

Reply:
Don't worry, my bad because I just copied)